pkg/names/names.go

```diff
@@ -47,25 +50,16 @@ func SetDeviceName(ifName string) string {
 
 	klog.V(4).Infof("Interface name '%s' is not DNS-1123 compliant, normalizing.", ifName)
 	encodedPayload := base32.StdEncoding.WithPadding(base32.NoPadding).EncodeToString([]byte(ifName))
-	normalizedName := NormalizedPrefix + strings.ToLower(encodedPayload)
+	normalizedName := NormalizedInterfacePrefix + strings.ToLower(encodedPayload)
 
 	return normalizedName
 }
 
-// GetOriginalName retrieves the original interface name from a deviceName.
-// If deviceName was prefixed with NormalizedPrefix (indicating it was encoded),
-// it decodes the name. Otherwise, it assumes deviceName is the original name.
-func GetOriginalName(deviceName string) string {
-	if strings.HasPrefix(deviceName, NormalizedPrefix) {
-		encodedPart := strings.TrimPrefix(deviceName, NormalizedPrefix)
-		encodedPart = strings.ToUpper(encodedPart) // base32 uses uppercase only
-		decodedBytes, err := base32.StdEncoding.WithPadding(base32.NoPadding).DecodeString(encodedPart)
-		if err != nil {
-			klog.Warningf("Failed to decode Base32 device name payload '%s' from full name '%s': %v. Returning the full deviceName as fallback.",
-				encodedPart, deviceName, err)
-			return deviceName
-		}
-		return string(decodedBytes)
-	}
-	return deviceName
+// NormalizePCIAddress takes a PCI address and converts it into a DNS-1123
+// acceptable format.
+func NormalizePCIAddress(pciAddress string) string {
```

Comment:
I can see how the restore pci address from the normalized name will be useful, as people or something will depend on that, can we make sure that we can recover the original address from the normalized one?